### ISSUE TITLE
Parsing booleans

### ISSUE THREAD

**Issue by maelp** [alexhallam/tv#48]
https://github.com/alexhallam/tv/blob/b548f0d19f64438d53f732fe2e98fad96a1ce178/src/datatype.rs#L10

Not sure if this is a mistake, but the function comments mentions scanning for "0" and "1" but they seem omitted from the regex

**Comment by alexhallam** [comment]
I need to revisit this. Thanks for pointing it out.

**Comment by alexhallam** [comment]
From regex101

I think this should work:

```
^true$|^false$|^t$|^f$|TRUE$|^FALSE$|^T$|^F$|^True|^False|^1$|^0$
```

![image](https://user-images.githubusercontent.com/9298693/135340651-c889ed6f-b132-48f0-9001-b1b1ddadb736.png)

**Comment by alexhallam** [comment]
This is open. All that needs to be changed is 

https://github.com/alexhallam/tv/blob/b548f0d19f64438d53f732fe2e98fad96a1ce178/src/datatype.rs#L10

to

```rust
Regex::new(r"^true$|^false$|^t$|^f$|TRUE$|^FALSE$|^T$|^F$|^True|^False|^1$|^0$").unwrap(); 
```

**Comment by alexhallam** [comment]
Closed with #61
